experiments/max_tool_per_agent/.env.example

```diff
@@ -0,0 +1,8 @@
+INFERENCE_MODEL="meta-llama/Llama-3.2-3B-Instruct"
+LLAMA_STACK_PORT=8321
+OLLAMA_URL=http://localhost:11434
+
+# Websearch tool related API Keys
+TAVILY_SEARCH_API_KEY="your_tavily_search_api_key"
```

Comment:
To you want to add a link to where you need to register your tavily API key, so that its consistent with the wolframAlpha note?